### ISSUE TITLE
Change tableName Comment to comment

### ISSUE THREAD

**Issue by rahulk-k** [yii2mod/yii2-comments#29]
```php

class CommentModel extends ActiveRecord
{
    /**
     * @var null|array|ActiveRecord[] Comment children
     */
    protected $_children;

    /**
     * Declares the name of the database table associated with this AR class.
     *
     * @return string the table name
     */
    public static function tableName()
    {
        return '{{%Comment}}'; // change Comment to comment
    }
?>
You should keep your table name as lower case. Because in Windows the case sensitive of table never change. Once you put the script in Linux it 'll change all to lower case.

**Comment by ihorchepurnyi** [comment]
We have plans about this, but now I don't have a time for all extensions.

**Comment by ihorchepurnyi** [comment]
Resolved by commit 11f3317
